File: Java/1283.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int a = sc.nextInt();
		int b = sc.nextInt();
		float res = a;
		float net;
		
		int per[] = new int[b];
		for(int i = 0; i < per.length; i++) {
			per[i] = sc.nextInt();
		}
		
		for(int i = 0; i < b; i++){
			res = res + res*(per[i]/(float)100);
		}
		
		net = res - a;
		
		System.out.printf("%.0f\n", net);
		
		if(-0.5 < net && net < 0.5) System.out.println("same");
		else if(net >= 0.5) System.out.println("good");
		else if(net <= -0.5) System.out.println("bad");
	}
}

File: Python/1283.py
a = int(input())
b = int(input())
res = float(a) ; net = 0

per = list(map(int, input().split()))

for i in range(b):
    res = res + res*(per[i]/100)

net = res - a

print("%.0f" % net)

if -0.5 < net < 0.5 : print("same")
elif net >= 0.5 : print("good")
elif net <= -0.5 : print("bad")